app/api/groups/groups.go

```diff
@@ -20,3 +22,15 @@ func (srv *Service) SetRoutes(router chi.Router) {
 	router.Get("/groups/{group_id}", service.AppHandler(srv.getGroup).ServeHTTP)
 	router.Put("/groups/{group_id}", service.AppHandler(srv.updateGroup).ServeHTTP)
 }
+
+func (srv *Service) checkThatUserOwnsTheGroup(user *auth.User, groupID int64) service.APIError {
```

Comment:
It cannot be a part of a store because it returns service.APIError. That's why it's here. Also, it is used only inside service/groups. I consider it as a helper method for service/groups.